devU-api/src/entities/containerAutoGrader/containerAutoGrader.validator.ts

```diff
@@ -1,19 +1,20 @@
-import { check } from 'express-validator'
+import {check} from 'express-validator'
 
 import validate from '../../middleware/validator/generic.validator'
 
 
-
 const assignmentId = check('assignmentId').isNumeric()
 
-const graderFile = check('graderFile').optional({ nullable: true }).custom(({ req }) => {
-    const file = req.files['grader']
+const graderFile = check('graderFile').custom(({req}) => {
+    const file = req?.files['grader']
     if (file !== null) {
         if (file.size <= 0) {
             throw new Error('File is empty')
         }
+    } else {
+        throw new Error('does not have grader file')
     }
-})
+}).withMessage('Grader file is required')
```

Comment:
Error message properly displays now, however now I get this error even when uploading a valid graderfile. Does this occur on your end when sending a request through the frontend?

Reply:
I dont believe so, i have tried it it passed, and returned to the last page